src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandlerTest.java

```diff
@@ -242,6 +242,19 @@ public void givenCaseWithEditedDwpDocsAndPhmeUnderReview_thenReturnErrorMessageA
         verifyNoInteractions(serviceRequestExecutor);
     }
 
+    @Test
+    public void givenCaseWithEditedDwpDocsAndChildSupport_thenReturnNoError() {
+        addMandatoryDwpDocuments();
+        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+        sscsCaseData.setLanguagePreferenceWelsh(NO.getValue());
+        sscsCaseData.getAppeal().setBenefitType(BenefitType.builder().code("childSupport").build());
+        sscsCaseData.setBenefitCode("022");
+
+        PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertEquals(0, response.getErrors().size());
```

Comment:
Is this actually a useful test? Shouldn't we check that the class actually hits the expected methods and gets the correct number of bundles when it is a Child Support case with edited docs?